src/app/globals.css

```diff
@@ -23,7 +23,8 @@
 html,
 body {
   max-width: 100vw;
-  overflow: hidden;
+  /* overflow: hidden; */
+  /* overflow-y: scroll; */
```

Comment:
why? Catch some problem with the layout?

Reply:
in categories page button at the bottom were not reachable, the page didn't scroll

Reply:
ok, thanks! Looks like scroll on pages works fine, no need to leave comments